src/cryptoadvance/specter/wallet.py

```diff
@@ -1562,6 +1562,10 @@ def get_address_name(self, address, addr_idx):
         # TODO: remove
         return self.getlabel(address)
 
+    @property
+    def rpc_folder(self):
```

Comment:
please add docstring

Reply:
With moving the unloading of the wallet on the Bitcoin Core side back to the wallet manager, we don't need that property on the wallet side at all anymore. Will remove it.